src/cascadia/TerminalApp/CommandPalette.cpp

```diff
@@ -364,6 +375,19 @@ namespace winrt::TerminalApp::implementation
             _searchBox().PasteFromClipboard();
             e.Handled(true);
         }
+        else if (key == VirtualKey::Right)
+        {
+            if (_currentMode == CommandPaletteMode::CommandlineMode)
+            {
+                if (const auto command{ _filteredActionsView().SelectedItem().try_as<winrt::TerminalApp::FilteredCommand>() })
+                {
+                    _searchBox().Text(command.Item().Name());
+                    _searchBox().Select(_searchBox().Text().size(), 0);
+                    _searchBox().Focus(FocusState::Programmatic);
+                }
+            }
+            e.Handled(true);
```

Comment:
probably want to move this into the if block. Can you still move in the text box with left/right arrow keys after this change to edit what you're typing?

Reply:
Good point! Thanks! Fixed.